### PR TITLE
Homepage redesign

### PR DESCRIPTION
**Pull request by KlemKlem**
Fast changes, TBC
Screenshot 2020-10-08 at 14 50 07
These are the things that should be immediately visible for the user:

- Logo
- Buttons (Lab / Game)
- Oneliner
- Short description
- Board

Maybe even the board should be above the description.

**Few other things:**

- version 2.2..... - I moved this down, this should not be the most important information
- testimonials - I tried doing something with them but had some problems (@Frizi happy to talk about this)
- the QG logo - It looks like on bigger screens it'll get to close to the next item (here the buttons)
- SM - I would add social media icons somewhere at the upper part of the page.

**Next steps:**

- visually dividing the homepage into separate parts, blocks
- less text more visual

### CODE REVIEW

File: src/components/HomePage/index.vue [hunk 1]
```diff
@@ -19,9 +12,18 @@
       </router-link>
     </div>
     <div class="hello" layout="column u10">
-      <h1>{{ i18n.t('homepage.main') }}</h1>
-      <p>{{ i18n.t('homepage.desc') }}</p>
+      <h2>{{ i18n.t('homepage.main') }}</h2>
+      <h1>{{ i18n.t('homepage.desc') }}</h1>
```
Comment (Frizi) [inline src/components/HomePage/index.vue:16]: `<h1>` is the top-level header, while `<h2>` is secondary. Also why can't the `desc` stay `<p>` here?

File: src/components/HomePage/index.vue [hunk 2]
```diff
@@ -19,9 +12,18 @@
...
       <homepage-interactive />
+    </div>
+    <div class="gameVersion">
```
Comment (Frizi) [inline src/components/HomePage/index.vue:19]: I think this version section should either be at the top as it was, or in the footer. Having it in in the middle of the above-the-fold section in it's own section probably gives it a little bit too much attention.

File: src/components/HomePage/HomepageTestimonials.vue [hunk 3]
```diff
@@ -17,14 +17,16 @@
...
     </div>
-    <p>
-      Cited in
-      <a
-        href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C5&amp;q=%22quantumgame.io%22+OR+%22Quantum+Game+with+Photons%22&amp;btnG="
-        target="_blank"
-        >numerous academic publications</a
-      >, the Top Pick for Education in Gamyfing Quantum Theory, mentioned in VICE and Boing Boing.
-    </p>
+    <div class="extra-text">
```
Comment (Frizi) [inline src/components/HomePage/HomepageTestimonials.vue:20]: that extra div is probably unnecessary. Class could go directly on `<p>` element

File: src/components/HomePage/HomepageTestimonials.vue [hunk 4]
```diff
@@ -76,22 +78,24 @@ export default defineComponent({
...
-  color: rgb(255, 181, 209);
+  color: #eab5ff;
   text-decoration: none;
 }
 a:hover {
   color: white;
 }
 
 .testimonial {
+  // background: rgba(255, 255, 255, 0.479);
```
Comment (Frizi) [inline src/components/HomePage/HomepageTestimonials.vue:89]: left commented out code

File: src/components/HomePage/index.vue (hunk 1 above)
Comment (KlemKlem) [reply src/components/HomePage/index.vue:16]: added `<p>` with `class`

File: src/components/HomePage/HomepageTestimonials.vue (hunk 3 above)
Comment (KlemKlem) [reply src/components/HomePage/HomepageTestimonials.vue:20]: ok

File: src/components/HomePage/HomepageTestimonials.vue (hunk 4 above)
Comment (KlemKlem) [reply src/components/HomePage/HomepageTestimonials.vue:87]: background only for the purpose of illustrating the problem - The testimonial fields overlap, how to change this? 
Screenshot 2020-10-09 at 21 55 24

File: src/components/HomePage/index.vue (hunk 2 above)
Comment (KlemKlem) [reply src/components/HomePage/index.vue:19]: too prominent for the top, I want users to see the two **buttons**, **the description**, and at least a bit of **the board**. 
I am happy to send it to the bottom, @stared ?

Comment (stared) [reply src/components/HomePage/index.vue:19]: When it comes to the commit info/date - well, it serves a few purposes:

* checking which version is that, for us (can be anywhere); useful, as sometimes the version we see is cached, not the newest
* giving hint that we continuously work on it (not e.g. abandoned in 2016) AND that it is a very alpha version so to set the expectations

Given these constraints, it can be anywhere (@KlemKlem, it is up to you). For me, I prefer it small font near the play button, OR even better - on the LOGO line (upper right corner).

What I do like, is this explicit style, see e.g. https://nodejs.org/en/
Screenshot 2020-10-10 at 16 00 14

Comment (KlemKlem) [reply src/components/HomePage/index.vue:19]: @stared I understand but our text is much longer than the example. Can we cut it to "Version 2.2.14 (beta)"? Or even "Version 2.2.14 (beta). Mobile version coming soon." Where the version is a link to GitHub?
I don't know how to add it to the right corner, but that could work.

Comment (stared) [reply src/components/HomePage/index.vue:19]: Let's call what it is (alpha). I kind of prefer dates than numbers, if we have to choose.

A link to GitHub is not necessary.

Comment (KlemKlem) [reply src/components/HomePage/index.vue:19]: However you prefer, I meant the length. But anyhow I don't know how to add it to the right corner.

Comment (Frizi) [reply src/components/HomePage/index.vue:19]: I personally prefer to just use a commit hash. `Version 2.2.14 (alpha build 119e83a)`

Dates are verbose and hard to parse, their format is language specific, and it doesn't really tell what it is in the end of the day. The hash also sets the expectation for a github link.

File: src/components/HomePage/HomepageTestimonials.vue (hunk 4 above)
Comment (Frizi) [reply src/components/HomePage/HomepageTestimonials.vue:87]: This is an unfortunate side effect of the layout system. The blocks that define `layout` might have negative margins, so adding background and borders to them can have results like this.

This particular layout is also a bit wrong, as it was created just before i adjusted the layout for wrapping.

If you want to style the whole section, you should target the outer `.testimonials` div, not individual cards.

**Review by Frizi — APPROVED** [review]
